packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -17,6 +21,7 @@ $resizerColor: $ms-color-neutralTertiaryAlt;
   border-bottom: 1px solid $dividerColor;
   cursor: default;
   user-select: none;
+  z-index: $ms-zIndex-front;
```

Comment:
Sure, I can change this.  Wasn't sure if it was acceptable or not to use non-variables for our z-indices.